clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/manifest/KubernetesPatchManifestOperation.java

```diff
@@ -56,6 +61,8 @@ private static Task getTask() {
     return TaskRepository.threadLocalTask.get();
   }
 
+  private static ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Any reason to use a new `ObjectMapper` rather than autowire the pre-configured one?

For example, here:
https://github.com/spinnaker/clouddriver/blob/29c5c1435ab0c750c4b491208f600647a3e98901/clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2ArtifactProvider.java#L40

Reply:
Updated!